mono/metadata/object.c

```diff
@@ -3931,6 +3932,12 @@ mono_nullable_box (guint8 *buf, MonoClass *klass, MonoError *error)
 		return NULL;
 }
 
+MonoObjectHandle
+mono_nullable_box_handle (gpointer buf, MonoClass *klass, MonoError *error)
```

Comment:
If I'm reading the pointer arithmetic in `mono_nullable_box` correctly, `buf` here points at the beginning of a managed object.  So we're depending on the callers to to use handles correctly and then pass a raw pointer here.

Reply:
Could be. Could be a half measure here and will need more attention soon.

Reply:
Buf will maybe become also a handle, and then use raw in here. But let's move ahead asis? I can only traverse the callgraph so much at a time..

Reply:
Let's add a FIXME so we know to come back here.

Reply:
Sure